tests/Mock/FosUser/MailerMock.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Tests\Mock;
+namespace ApiPlatform\Core\Tests\Mock\FosUser;
```

Comment:
You have to update this too: https://github.com/api-platform/core/blob/0bbf5c9154497983e6bff2a0e9bc81ef9a370f4e/tests/Fixtures/app/config/config.yml#L90